docs/releasing-files.md

```diff
@@ -165,16 +165,37 @@ If you are likely to release data multiple times, e.g. for initial discussion wi
 
 ## 2. Requesting release of outputs from the server
 
-Having applied disclosure controls to the aggregated study data, you are ready to request their release. **Only specific members of the OpenSAFELY team trained in output checking have permissions to release the data**. When you are ready to request a release of your aggregated results, and only the minimum outputs necessary please [complete this form](documents/OpenSAFELY_Output_Review_Form_07-04-22.docx) and email us at **<datarelease@opensafely.org>**. For each output wishing to be released you will need to provide a clear contextual description including:
+Having applied disclosure controls to the aggregated study data, you are ready to request their release. **Only specific members of the OpenSAFELY team trained in output checking have permissions to release the data**. When you are ready to request a release of your aggregated results, and only the minimum outputs necessary please [complete this form](documents/OpenSAFELY_Output_Review_Form_07-04-22.docx). For each output wishing to be released you will need to provide a clear contextual description including:
```

Comment:
what's the rationale for removing this email address?

Reply:
I've moved this into a section below on submitting the form to separate out filling out the form and what it should include, from the submission process